docs/DeveloperGuide.md

```diff
@@ -1066,54 +1066,116 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### 6.3 Use cases
 
-(For all use cases below, the **System** is `PatientSync` and the **Actor** is the `nurse`, unless specified otherwise)
+(For all use cases below, the **System** is `PatientSync` and the **Actor** is the `Nurse`, unless specified otherwise)
 
-**Use case: Add a patient**
+**Use case: UC01 - Add a patient**
 
 **MSS**
 
-1.  Nurse requests to add a patient
-2.  PatientSync adds the patient
+1. Nurse requests to add a patient.
+1. PatientSync adds the patient and display success message.
 
     Use case ends.
 
 **Extensions**
 
-* 2a. The information key in is wrong.
-    * 2a1. PatientSync shows an error message.
+* 1a. Nurse's input is invalid command format.
 
+    * 1a1. PatientSync shows an error message.
+           
       Use case ends.
+  
+* 1b. Nurse's input contains invalid parameters value. 
 
+    * 1b1. PatientSync shows an error message. 
+    
+      Use case ends.
 
-**Use case: Delete a patient**
+**Use case: UC02 - Delete a patient**
 
 **MSS**
 
-1.  Nurse requests to list patients
-2.  PatientSync shows a list of patients
-3.  Nurse requests to delete a specific patient in the list
-4.  PatientSync deletes the patient
+1. Nurse requests to list patients.
+1. PatientSync shows a list of patients.
+1. Nurse requests to delete a specific patient in the list.
+1. PatientSync deletes the patient and displays success message.
 
     Use case ends.
 
 **Extensions**
 
 * 2a. The list is empty.
+      Use case ends.
+
+* 3a. Nurse's input is invalid command format.
+
+    * 3a1. PatientSync shows an error message.
+    
+      Use case ends.
+
+* 3b. Nurse's input contains invalid patient id.
 
+    * 3b1. PatientSync shows an error message.
+    
+      Use case ends.
+
+**Use case: UC03 - Edit a patient**
+
+**MSS**
+
+1. Nurse requests to list patients.
+1. PatientSync shows a list of patients.
+1. Nurse requests to edit a specific patient in the list.
+1. PatientSync edit the patient and displays success message.
+
+   Use case ends.
+
+**Extensions**
+
+* 2a. The list is empty.
   Use case ends.
 
-* 3a. The given index is invalid.
+* 3a. Nurse's input is invalid command format.
 
-    * 3a1. PatientSync shows an error message.
+  * 3a1. PatientSync shows an error message.
+
+    Use case ends.
+
+* 3b. Nurse's input contains invalid patient id.
```

Comment:
Would suggest to change -> Nurse's input contains invalid ~~patient id~~ `patientHospitalId` / `patient's hospital ID` (depending on the other use cases format)

Reply:
the patient ID here should be referring to the id in the patient list, thus not `patientHospitalId`

Reply:
Would it be better if we use index instead?